data-prepper-expression/build.gradle

```diff
@@ -13,7 +13,9 @@ ext {
 }
 
 dependencies {
-    antlr 'org.antlr:antlr4:4.10.1'
+    antlr('org.antlr:antlr4:4.10.1') {
+        exclude group: 'org.glassfish', module: 'javax.json'
+    }
```

Comment:
Exclude `javax.json` to fix a `java.util.ServiceConfigurationError: jakarta.json.spi.JsonProvider: org.glassfish.json.JsonProviderImpl not a subtype` error when running Data Prepper. This error shows up in the dev/trace-analytics-sample-app demo.

`javax.json` is a transitive dependency through `antlr4`'s Tool module which I don't see used in Data Prepper.

Before the change, we have both `jakarta.json` and `javax.json` as dependencies and they both implement similar classes under `org.glassfish.json` package name, which is likely the cause of the issue.